cumulusci/core/config/org_config.py

```diff
@@ -31,6 +33,39 @@
 class OrgConfig(BaseConfig):
     """Salesforce org configuration (i.e. org credentials)"""
 
+    access_token: str
+    config_file: str
+    config_name: str
+    created: str
+    date_created: str
+    days: int
+    email_address: str
+    instance_name: str
+    instance_url: str
+    is_sandbox: str
+    namespace: str
+    namespaced: str
+    org_id: str
+    org_type: str
+    password: str
+    scratch: str
+    scratch_org_type: str
+    set_password: str
+    sfdx_alias: str
+    username: str
+    userinfo: str
+    id: str
+    expires: str
+    active: str
+    default: str
+    expired: bool
+    client_id: str
+    refresh_token: str
+    client_secret: str
+    connected_app: str
```

Comment:
Some of these are valid/expected for ScratchOrgConfigs but not OrgConfigs. Some of the types look wrong. And really an OrgConfig may have any attribute that was returned by the oauth auth flow.

Reply:
Also I think it'll lead to confusion that some of these are expected to be None sometimes but are not declared as Optional